2.1/alpine/Dockerfile

```diff
@@ -78,6 +78,7 @@ RUN set -ex \
 		scanelf --needed --nobanner --recursive /usr/local \
 			| awk '{ gsub(/,/, "\nso:", $2); print "so:" $2 }' \
```

Comment:
Yes, and we have two libruby.so installed.

`grep -v libruby` removes `ruby-libs` from dependencies list. My solution is only a workaround and I would like to point at the issue.